index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone</title>
+    <link rel="stylesheet" href="style.css">
+    <script
+      src="https://kit.fontawesome.com/5d13e94c8b.js"
+      crossorigin="anonymous"
+    ></script>
+</head>
+<body>
+    <header class="header">
+        <div class="header-div">
+            <div>
+            <ul class="icons-ul">                
+                <li><img class="head-icon" src="./images/faceb.png" alt="facebookicon"></li>
+                <li><img class="head-icon" src="./images/twtr.png" alt="twitter-icon"></li>
+                <li class="link">Other language</li>
+                <li class="link">my page</li>
+                <li class="link">Logout</li>
+            </ul>
+            </div>
+            
+            <div class="bars">
+        </div>
+                 <nav>
+                    <div class="span-"><img  src="./images/download.jpg" alt="#" class="inner-backg"><p class="logo">LIVING FAITH CHURCH</p> </div>
+
+                    <ul class="nav-ul">
+                        <li><a href="about.html">About</a></li>
+                        <li><a href="index.html">Home</a></li>
+                        <li>Program</li>
+                        <li>Join</li>
+                        <li>Sponsor</li>
+                        <li>News</li>
+                        <li><h3 class="winners">Winners Chapel</h3></li>
+                    </ul>
+                </nav>
+            </div>
+    </header>
+    
+    <nav>
+        <article class="intro">
+            <div class="menu-nav">
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+                <div class="hamburger"></div>
+          </div>
+
+           
+<div class="mobileMenu"> 
+  
+<div class="m1"><h2 class="m2">Winners Chapel</h2></div>
+
+<span class="closeMenu">&times;</span>
+  <ul class="container">
+        <li><a href="about.html">About</a></li>
+        <li><a href="index.html">Home</a></li>
+        <li>Program</li>
+        <li>Join</li>
+        <li>News</li>
+ </ul>
+
+</div>
+        </article>
+
+    </nav>
+
+    <main>
+        
+<div class="pg1">
+    <h2 class="theme">
+        "Youth For Christ"
+     </h2>
+     <h1 class="ch-name">Winners Chapel Youth Conference 2022.</h1>
+    <div class="moment">
+         <p class="prg">in His presnce there is fullness of joy, experience God in another demention come youth 2022 conference</p>
+    </div>
+     <div class="info">
+        <h6 class="date">22.05.2022(THUR)~23(FRI)</h6>
+        <p class="venue">@ Church Auditorium, Lagos.</p>
+    </div>
+ </div>
+
+ <section class="prog-card">
```

Comment:
- [ ] As the previous reviewer requested, in the mobile view these cards overflow many screen sizes.  The reason is that the elements are using fixed properties.  

## Explanation

- Fixed properties measurements are the ones like `px`.  They are fixed no matter the screen size.
- Dynamic properties are ones like `rem`, `%`.  These take into account the screen size.
- You can't know the screen size of your users, that is why it is displaying great for you, and bad for others.  Not everyone will have the same screen size.  This is especially true for mobile.

##  Hints

- In the `prog-card` class, you have a `width: 461px`.  This is fixed, so it's causing problems.  Use flex instead like justify-content, align-items, etc.  OR  use something like `width: 30%` etc  but not a fixed measurement.
- In `card-section` you have a `margin-left:30px`.  This is bad practice because it causes trouble with the different screen sizes.  I would remove the margin and just use `align-items: center`.
- In the `join` class, instead of a `margin-left: 54px` (bad practice), use flex instead to align that box.  `margin: auto` will take care of the rest:

```css
display: flex;
justify-content: center:
margin: auto;
```
Result:

![image](https://user-images.githubusercontent.com/57421823/164949283-31ac47c5-7eb8-416f-9ebb-852230c64162.png)

Please, do this because it is a super important skill to be able to align and center elements at all screen sizes, without them overflowing.  

Also, even if I provided you with tips and a simple solution, please remember it.  This is not the last time you will have to do this properly for all screen sizes.

You will use this in the future, trust me.  💪 

_Note: I wrote this as a result of my own experience and it is not to be taken as an "official" way to do it.  This is just one way of many to achieve the objective._

Reply:
done

Reply:
done